pyproject.toml

```
@@ -29,12 +29,12 @@ dynamic = ["version"]
 dependencies = [
   "aicspylibczi>=3.1.1",
```

might as well....assuming the tests pass
```suggestion
  "aicspylibczi>=3.2.0",
```